lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/ConfigurationManager.cs

```diff
@@ -115,6 +119,17 @@ internal sealed class ConfigurationManager
             {
                 Exceptions.Throw(e, "Unable to deserialize map input pipeline data converter configuration", Logger);
             }
+
+            try
+            {
+                _resultHandlerConfiguration =
+                    configurationSerializer.FromString(resultHandlerConfiguration);
+                Logger.Log(Level.Info,"Serialized result handler is " + resultHandlerConfiguration);
+            }
+            catch (Exception e)
+            {
+                Exceptions.Throw(e, "Unable to deserialize map input pipeline data converter configuration", Logger);
```

Comment:
I believe this should be `ReThrow()`, but the exact semantics of the `Exceptions` class still mystify me.